### PR TITLE
feat(wallet-mobile): Enable governance on preprod network

### PR DESCRIPTION
**Pull request by michalszorad**
## Description / Change(s) / Related issue(s)

Enable governance on preprod network

##### Ticket

[YOMO-1872](https://emurgo.atlassian.net/browse/YOMO-1872)

[YOMO-1872]: https://emurgo.atlassian.net/browse/YOMO-1872?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/kernel/config.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const governaceAfterBlock = freeze({
 
   // TODO: Add block number once known
   [Chain.Network.Mainnet]: Infinity,
-  [Chain.Network.Preprod]: Infinity,
+  [Chain.Network.Preprod]: 2639667,
```
Comment (michalszorad) [inline apps/wallet-mobile/src/kernel/config.ts:11]: 30 Aug 2024 07:15 UTC time 
https://preprod.cexplorer.io/block/ba5ea7947919c0522741c67a905be569e0269c65beee0e3413f78b69b130354f

Comment (stackchain) [reply apps/wallet-mobile/src/kernel/config.ts:11]: @michaeljscript I think it can be applied to `mainnet` as well, after Sep 1st.

Comment (stackchain) [reply apps/wallet-mobile/src/kernel/config.ts:11]: so it "doesn't even need to check the network" anymore

Comment (michalszorad) [reply apps/wallet-mobile/src/kernel/config.ts:11]: Should we do it in a different PR once we're sure about the merge or in this PR?

**Review by stackchain — APPROVED** [review]

File: apps/wallet-mobile/src/kernel/config.ts (hunk 1 above)
Comment (stackchain) [reply apps/wallet-mobile/src/kernel/config.ts:11]: create another one

**Pull request merged**